ddtrace/bootstrap/preload.py

```diff
@@ -104,6 +104,13 @@ def _():
 
 if "DD_TRACE_GLOBAL_TAGS" in os.environ:
     env_tags = os.getenv("DD_TRACE_GLOBAL_TAGS")
+    if env_tags is not None:
+        deprecate(
+            "DD_TRACE_GLOBAL_TAGS is deprecated",
+            message="DD_TRACE_GLOBAL_TAGS is deprecated, please migrate to using DD_TAGS instead",
```

Comment:
This will show up redundantly like:
```
DD_TRACE_GLOBAL_TAGS is deprecated: DD_TRACE_GLOBAL_TAGS is deprecated, please migrate to using DD_TAGS instead
```
Can we remove it from the second message string like:
```suggestion
            message="Please migrate to using DD_TAGS instead",
```